docs/README.md

```diff
@@ -5,56 +5,52 @@
 Vue InstantSearch is the ultimate toolbox for creating instant-search
 experiences using [Vue.js](https://vuejs.org/) and [Algolia](https://www.algolia.com/).
 
-## Algolia demo credentials
-
-To be able to use Vue InstantSearch you will need an Algolia account. Once you have an Algolia account, you can get
-the credentials from the [Algolia Dashboard](https://www.algolia.com/api-keys).
-
-For the sake of simplicity, here are some demo credentials that will give you access to
-an e-commerce website's index.
-
- - `appId`: `latency`
- - `searchKey`: `3d9875e51fbd20c7754e65422f7ce5e1`
- - `indexName`: `bestbuy`
-
 ## Setup a new Vue project
 
-You can bootstrap a new Vue project with the [official CLI tool](https://vuejs.org/v2/guide/installation.html#CLI).
```

Comment:
I removed all "You can" "You should" to go for action/present sentences (this is a getting started tutorial, we must be directive so there's no place for the user to wonder/ask questions).